sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -522,6 +610,16 @@ private void processSendWork() {
                 if (workItem.isDeliveryStateProvided()) {
                     delivery.disposition(workItem.getDeliveryState());
                 }
+
+                if (workItem.getMessage() != null) {
```

Comment:
is it still possible after this change?

Reply:
Once we finalize, this method will no longer return byte[] but ReadableBuffer